Add a new descritpion for the DataGrid - Filter API demo (#75)
DevExpress/devextreme-demos: JSDemos/Demos/DataGrid/FilteringAPI/description.md

> **DataGrid** has the following API that you can use to filter data:     

Reviewer: "to have" is a weak verb and I'd always encourage to find a replacement. I'd probably say "includes".

```suggestion
**DataGrid** includes the following API that you can use to filter data:     
```

> **DataGrid** includes the following API that you can use to filter data:     

Reviewer 2: ```suggestion
The **DataGrid** includes the following API you can use to filter data:     
```

- [filter()](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Methods/#filter) / [filter(filterExpr)](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Methods/#filterfilterExpr)       

> Gets or applies a filter expression to the grid's [dataSource](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#dataSource).

Reviewer: It may be read like "gets a filter expression to the grid's... ". I'd use "specifies" or "sets" to avoid this conjunction issue: 

```suggestion
Specifies a filter expression for the grid's [dataSource](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#dataSource).
```

> Gets the filter expression applied to the grid's [dataSource](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#dataSource). If you specify the `filterExpr` parameter, this method applies the filter expression to the grid's **dataSource**.

Reviewer 3: ```suggestion
Gets or applies a filter expression to the grid's [dataSource](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#dataSource).
```

- [filterValue](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterValue)       

> A filter expression applied using the following UI elements: [filter row](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/filterRow/), [header filter](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/headerFilter/), [filter builder](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterBuilder).

Reviewer 2: ```suggestion
A filter expression is applied with the following UI elements: [filter row](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/filterRow/), [header filter](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/headerFilter/), [filter builder](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterBuilder).
```

Author: Here we describe **what** this option is, e.g. we should say something like "A filter expression that is applied with the..."

Reviewer 2: Ok

> Specifies a filter expression. Includes filters from the following UI elements: [filter row](/Documentation/Guide/Widgets/DataGrid/Filtering_and_Searching/#Filter_Row), [header filter](/Documentation/Guide/Widgets/DataGrid/Filtering_and_Searching/#Header_Filter), and [filter builder](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterBuilder).

Reviewer 3: ```suggestion
A filter expression applied using the following UI elements: [filter row](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/filterRow/), [header filter](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/headerFilter/), [filter builder](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterBuilder).
```

> - [getCombinedFilter()](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxDataGrid/Methods/#getCombinedFilter) / [getCombinedFilter(returnDataField)](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Methods/#getCombinedFilterreturnDataField)       

Reviewer 3: ```suggestion
- [getCombinedFilter()](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Methods/#getCombinedFilter) / [getCombinedFilter(returnDataField)](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Methods/#getCombinedFilterreturnDataField)       
```

> Gets the total filter that includes filters from the UI elements and the filter applied to the **dataSource**. If you specify the `returnDataField` parameter, the return value will contain data field names instead of [getters](/Documentation/Guide/Data_Binding/Data_Layer/#Getters_And_Setters).

Reviewer 2: ```suggestion
Gets the total filter that includes filters from the UI elements and the filter applied to the **dataSource**. If you specify the `returnDataField` parameter, the returned value contains data field names instead of [getters](/Documentation/Guide/Data_Binding/Data_Layer/#Getters_And_Setters).
```

> Gets the total filter that includes filters from the UI elements and **filter(filterExpr)** method. If you specify the `returnDataField` parameter, the return value will contain data fields instead of [getters](/Documentation/Guide/Data_Binding/Data_Layer/#Getters_And_Setters).

Reviewer 3: ```suggestion
Gets the total filter that includes filters from the UI elements and the filter applied to the **dataSource**. If you specify the `returnDataField` parameter, the return value will contain data field names instead of [getters](/Documentation/Guide/Data_Binding/Data_Layer/#Getters_And_Setters).
```

- [clearFilter()](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Methods/#clearFilter) / [clearFilter(filterName)](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Methods/#clearFilterfilterName)       
Discards all filters applied to the widget. If you specify the `filterName` parameter, this method discards all filters of a specific type.

> In this demo, you can use the [SelectBox](/Documentation/ApiReference/UI_Widgets/dxSelectBox/) widget to filter the grid's **dataSource**. When you select an item from the **SelectBox**, the **filter(filterExpr)** or **clearFilter()** method is called, depending on your selection.

Reviewer 2: ```suggestion
In this demo, you can use the [SelectBox](/Documentation/ApiReference/UI_Widgets/dxSelectBox/) widget to filter the grid's **dataSource**. When you select an item from the **SelectBox**, the **filter(filterExpr)** or **clearFilter()** method is called.
```

Reviewer 2: Why add 'depending on your selection' if the selection options are known?

Author: The reason is we have six selection options, and only one of them calls the **clearFilter()** method. Other selection options call the **filter(filterExpr)** method is called. Do you think that it will be clear for customers without the phrase that you suggest to omit?

Reviewer 2: Is it clear from the names which method does what? For instance, if it is called 'clear' something, you know it corresponds to the clearFilter() method.

Author: No, it isn't clear and this is the problem. The selection options are the following:
![image](https://user-images.githubusercontent.com/33040934/92885310-507ec100-f41b-11ea-861f-f8e5f01c36b3.png)
Only the first option ('All') calls the **clearFilter()** method. Other options call the **filter(filterExpr)** method.

Reviewer 2: Ok, but then you should say that: In this demo, you can use the [SelectBox](/Documentation/ApiReference/UI_Widgets/dxSelectBox/) widget to filter the grid's **dataSource**. 'All' calls the clearFilter() method and the other items call the **filter(filterExpr)**method.

> In this demo, the grid's **dataSource** filter settings are synchronized with the [SelectBox](/Documentation/ApiReference/UI_Widgets/dxSelectBox/) value. Select an option in a drop-down list to call **filter(filterExpr)** or **clearFilter()** methods depending on the selection.

Reviewer 3: ```suggestion
In this demo, you can use the [SelectBox](/Documentation/ApiReference/UI_Widgets/dxSelectBox/) widget to filter the grid's **dataSource**. When you select an item from the **SelectBox**, the **filter(filterExpr)** or **clearFilter()** method is called, depending on your selection.
```
